encode_utils/connection.py

```diff
@@ -1212,13 +1212,13 @@ def remove_props(self, rec_id, props=[]):
             prop = profile.get_property_from_name(prop_name)
             if prop.is_required:
                 raise Exception("Can't remove required property")
-            elif prop.is_not_submittable:
+            elif prop.is_not_submittable():
                 raise Exception("Can't remove non-submittable property.")
-            elif prop.is_read_only:
+            elif prop.is_read_only():
```

Comment:
Similar to above. Won't need this change if we make `is_read_only` into an `@property`:

https://github.com/StanfordBioinformatics/encode_utils/blob/4cde56ff9418177bf4297c4ec0b6695f0e4eb623/encode_utils/profiles.py#L201